Makefile

```diff
@@ -125,6 +124,7 @@ generate: gen-helm-docs ## Generate manifests, code, and docs resources
 
 .PHONY: deploy
 deploy: ## Deploy manager resources to the K8s cluster
+	kind load docker-image $(IMG) --name $(TEST_KIND_CLUSTER)
```

Comment:
Any specific reason why this is part of deploy now? If this is required for testing/local purposes, then it can be removed from here.

Reply:
You are right this is not the best place for it, the previous one was equally bad though, so I'll remove it from here and add this line to the readme under the instructions...

Reply:
That would be okay. You can use additional (hidden) target just for pushing image to kind.